spec/internal/app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User
   include Mongoid::Document
+  include Matterhorn::Inclusions::InclusionSupport
```

Comment:
add_inclusion requires that on class. but the issue is not that I actually need inclusion. The problem is this line throws exception even though I'm not using the inclusion for a resource :index or :show: 

https://github.com/blasterpal/matterhorn/blob/27_multiid_get/lib/matterhorn/serialization/scoped.rb#L68

Without including the module the resource doesn't have `__inclusion_configs`. Could we possibly change this code, yes. Do you think we should open an issue and refactor this? I presumed ` Matterhorn::Inclusions::InclusionSupport` was required for models.